File: QolmsYappli/QolmsYappli/Models/InnerModel/AmazonGiftCardItem.vb
''' <summary>
''' Amazonギフト券情報を表します。
''' このクラスは継承できません。
''' </summary>
''' <remarks></remarks>
<Serializable()>
Public NotInheritable Class AmazonGiftCardItem

#Region "Public Property"

    ''' <summary>
    ''' クーポン種別を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property GiftCardType As Byte = Byte.MinValue

    ''' <summary>
    ''' 消費ポイントを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property DemandPoint As Integer = Integer.MinValue

    ''' <summary>
    ''' 表示名を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property GiftCardName As String = String.Empty

    ''' <summary>
    ''' 残り枚数のカウントを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property RestCount As Integer = Integer.MinValue

#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="AmazonGiftCardItem" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
    End Sub

#End Region

End Class

File: QolmsJotoWebView/Models/InnerModels/AmazonGiftCardItem.cs
using System;

namespace MGF.QOLMS.QolmsJotoWebView
{
    /// <summary>
    /// Amazonギフト券情報を表します。
    /// このクラスは継承できません。
    /// </summary>
    [Serializable]
    public sealed class AmazonGiftCardItem
    {
        #region Public Property

        /// <summary>
        /// クーポン種別を取得または設定します。
        /// </summary>
        public byte GiftCardType { get; set; } = byte.MinValue;

        /// <summary>
        /// 消費ポイントを取得または設定します。
        /// </summary>
        public int DemandPoint { get; set; } = int.MinValue;

        /// <summary>
        /// 表示名を取得または設定します。
        /// </summary>
        public string GiftCardName { get; set; } = string.Empty;

        #endregion

        #region Constructor

        /// <summary>
        /// <see cref="AmazonGiftCardItem" /> クラスの新しいインスタンスを初期化します。
        /// </summary>
        public AmazonGiftCardItem()
        {
        }

        #endregion
    }
}
